### ISSUE TITLE
\x auto: list index out of range 

### ISSUE THREAD

**Issue by carymrobbins** [dbcli/pgcli#417]
It seems that empty result sets with `\x auto` enabled cause a `list index out of range` exception.

```
crobbins> select 1 where false;         
+------------+                          
| ?column?   |                          
|------------|                          
+------------+                          
SELECT 0                                       
          
crobbins> \x                            
Expanded display is on.                            

crobbins> select 1 where false;         
                                        
SELECT 0                                
             
crobbins> \x auto                       
Expanded display is used automatically. 
                 
crobbins> select 1 where false;         
list index out of range                 
```

**Comment by amjith** [comment]
Thanks for reporting the bug. It is indeed an oversight. I've created a PR (#419) to fix it. It should be fixed in the next release.

**Comment by amjith** [comment]
This is now available in v1.0. Please upgrade. 

```
pip install -U pgcli
```
